src/components/RecipeList.tsx

```diff
@@ -0,0 +1,22 @@
+import  { FC } from 'react';
+import RecipeCard from '../components/RecipeCard';
+import { Recipe } from '../utilities/utility';
+
+interface RecipeListProps {
```

Comment:
Since we have single property, can we call it as `RecipeListProp` ?
```suggestion
interface RecipeListProp {
```

Reply:
fixed https://github.com/Sakshi-Simform/Recipe-App/pull/3/commits/44f1054726f437623757a506493e4c783ab2aa00